files/weston.service

```diff
@@ -0,0 +1,8 @@
+[Unit]
+Description=Weston Compositor
+
+[Service]
+ExecStart=/usr/bin/weston --socket=waydroid --width=480 --height=1080
```

Comment:
I think the name of the service is misleading. Yes, Weston is the compositor here but it's launching Waydroid.

Reply:
ok i will rename it